docs/source/gdpr/data_user_agreement.rst

```diff
@@ -41,7 +41,7 @@ By accepting this agreement, I become the `data controller <https://ec.europa.eu
 1. I will comply with all relevant rules and regulations imposed by my institution and my government.  This agreement never has prevalence over existing general data protection regulations that are applicable in my country.
 2. I will not attempt to establish or retrieve the identity of the study participants. I will not link these data to any other database in a way that could provide identifying information. I shall not request the pseudonymisation key that would link these data to an individual's personal information, nor will I accept any additional information about individual participants under this Data Use Agreement.
 3. I will not redistribute these data or share access to these data with others, unless they have independently applied and been granted access to these data, i.e., signed this Data Use Agreement. This includes individuals in my institution.
-4. [OPTIONAL] When sharing secondary or derivative data (e.g. group statistical maps or templates), I will only do so if they are on a group level, and cannot be deduced information from individual participants.
+4. [OPTIONAL] When sharing secondary or derivative data (e.g. group statistical maps or templates), I will only do so if they are on a group level, and information from individual participants cannot be deduced.
```

Comment:
please increment version above (out of diff, can't recommend) so it becomes

```

**Version:** OBC-GDPR-DUA 1.0.1
```

Reply:
Version bumped to 1.0.1.